src/main/java/org/fcrepo/importexport/common/NullLogger.java

```diff
@@ -0,0 +1,396 @@
+/*
```

Comment:
Is this different from NOPLogger?  http://www.slf4j.org/api/org/slf4j/helpers/NOPLogger.html